DominicGBauer/information-access
# fix: bug squash for 150 integrate filter sidebar

# Pull Request

## Scope
https://dev.azure.com/JHNMAT015/ECO5037S%20Front%20end/_workitems/edit/150

## Work Done
* Added Options.vue to contain selection list
* Added alerts for incorrect custom filter entries eg: start date> end date or year not a number or negative values
* filter entries reset on search and if no custom input is present
* In mobile: on click away from drop down if there is no custom filter entries that have been applied (press go), dropdown resets

## Steps to Test
```npm run start```
check custom filter for both mobile and desktop: alerts appear and correct number of search results reflect

## Screenshot
![image](https://user-images.githubusercontent.com/80747408/138009764-15eb89bc-b466-426c-9dfa-6dcfdbd3aa55.png)
![image](https://user-images.githubusercontent.com/80747408/138009842-32ffe865-f8e5-46b8-a79c-2ba64c8601d7.png)
![image](https://user-images.githubusercontent.com/80747408/138009905-7c5ebefa-c51f-4833-82ad-8f9850f459a5.png)
